docs/technical-docs/git_and_github.md

```diff
@@ -56,6 +57,10 @@ For simplicity, the personal access token is recommended.  Authenticate by follo
 
 For details on other authentication options, check out [this resource page](https://docs.github.com/en/github/authenticating-to-github).
 
+## Notifications
+
+You can control whether GitHub sends you web notifications, e-mail notifications, or both when you are @mentioned inside GitHub. To ensure that you see these messages (especially when you are first learning GitHub), it is recommended that you select both "Email" and "Web and Mobile" under the Notifications tab of your Account Settings. In this way, when someone tags you on GitHub, you will receive an e-mail, and you will also be able to access the message from the :bell: icon in the top-right corner of the GitHub webpage.
```

Comment:
Maybe restructure the paragraph so first it explains what's the purpose and then how to do it:

You can control whether GitHub sends you web notifications, e-mail notifications, or both when you are @mentioned inside GitHub. In this way, when someone tags you on GitHub, you will receive an e-mail, and you will also be able to access the message from the :bell: icon in the top-right corner of the GitHub webpage.

- To ensure that you see these messages (especially when you are first learning GitHub), it is recommended that you select both "Email" and "Web and Mobile" under the Notifications tab of your Account Settings.

Reply:
@apoly1 This is a great idea!  Would you please pull this branch to your local (navigate to the wiki directory on your local > git checkout main > git pull > git checkout notifications-jess) and make the suggested changes in the .md file, add to the staging are, commit, then push back up to the repo?  You'll need to tag me on this convo after that's done so that I know you made the suggested changes. :)